user_docs/en/changes.t2t

```diff
@@ -8,9 +8,9 @@ What's New in NVDA
 == New Features ==
 - In Windows 11, NVDA will announce alerts from voice typing and suggested actions including the top suggestion when copying data such as phone numbers to the clipboard (Windows 11 2022 Update and later). (#16009, @josephsl)
 - Added support for the BrailleEdgeS2 braille device. (#16033)
+- Playing silence after every utterance in order to keep audio device open (#14386, @jcsteh, @mltony)

```

Comment:
Wouldn't you put something more user oriented?

E.g.:
```suggestion
- NVDA will play silence after every utterance in order to prevent the start of next speech to be clipped with some audio devices such as Bluetooth headphones. (#14386, @jcsteh, @mltony)
```

But this may probably be reworded still better.